components/openzwave-adapter/src/lib.rs

```diff
@@ -337,11 +366,15 @@ impl OpenzwaveAdapter {
                         // https://github.com/fxbox/taxonomy/issues/97
                         // When it's done we can move the properties change from above to here.
                     }
-                    ZWaveNotification::NodeRemoved(_node)           => {}
+                    ZWaveNotification::NodeRemoved(node)           => {
+                        if let Some(service_id) = node_map.find_taxo_id_from_ozw(&node) {
+                            let _ = box_manager.remove_service(&service_id);
```

Comment:
also use the result to display a log here; and we should remove the node from node_map